lib/appoptics_apm/version.rb

```diff
@@ -6,10 +6,10 @@ module AppOpticsAPM
   # The current version of the gem.  Used mainly by
   # appoptics_apm.gemspec during gem build process
   module Version
-    MAJOR = 4  # breaking,
-    MINOR = 12 # feature,
-    PATCH = 2  # fix => BFF
-    PRE   = nil
+    MAJOR = 0  # breaking,
+    MINOR = 1 # feature,
+    PATCH = 0  # fix => BFF
+    PRE   = "ghtest15"
```

Comment:
Minor, should reset to nil?

Reply:
The strategy will be to set the correct version when merging commits into master that will then be used to create a new gem 
I'll reset it to the current gem version